packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -4653,6 +4653,261 @@ curl -v -X GET \
 }
 ```
 
+## Application OAuth 2.0 Scope Consent Grant Operations
+
+<ApiLifecycle access="ea" />
+
+A scope consent Grant represents permission for the application to request that Okta scopes be included in OAuth 2.0 Bearer tokens. Without this Grant, token requests that contain Okta scopes are denied.
+
+### Grant Consent to Scope for Application
+
+<ApiLifecycle access="ea" />
+
+<ApiOperation method="post" url="/api/v1/apps/${applicationId}/grants" />
+
+Grants consent for the application to request OAuth 2.0 Okta scopes
+
+#### Request Parameters
+
+| Parameter       | Description                                                                                    | Param Type   | DataType   | Required   | Default |
+| :-------------- | :--------------------------------------------------------------------------------------------- | :----------- | :--------- | :--------- | :------ |
+| applicationId   | ID of the application                                                                          | URL          | String     | TRUE       |         |
+| issuer          | The issuer of your Org Authorization Server, your Org URL                                    | Body         | String     | TRUE       |         |
+| scopeId         | The name of the Okta scope for which consent is to be granted                                      | Body         | String     | TRUE       |         |
```

Comment:
two things:
1. if the param is `scopeId`, is it confusing to refer to it as the scope _name_?
2. `...Okta scope for which to grant consent`

Reply:
yes the parameter `scopeId` is definitely confusing and this was discussed a fair bit when this contract was being laid down. I think that it would be confusing to not specify that the string literal scope is what is taken here (what the heck is a scopeId without some clarification?), so I referred to that as "name". What phrasing would you suggest?

Reply:
Hm, I see what the problem is. I guess you could phrase it as `The identifier of the Okta scope...` 
What might be more helpful is including a link to https://developer.okta.com/docs/guides/implement-oauth-for-okta/scopes/ as "accepted values"

Reply:
I've linked to the scope list in the description of the field